collect_app/src/main/java/org/odk/collect/android/formentry/media/PromptAutoplayer.java

```diff
@@ -30,7 +31,7 @@ public PromptAutoplayer(AudioHelper audioHelper, ReferenceManager referenceManag
     }
 
     public Boolean autoplayIfNeeded(FormEntryPrompt prompt) {
-        String autoplayOption = prompt.getFormElement().getAdditionalAttribute(null, AUTOPLAY_ATTRIBUTE);
+        String autoplayOption = FormEntryPromptUtils.getBodyAttribute(prompt, AUTOPLAY_ATTRIBUTE);
```

Comment:
Maybe we should rename this helper to `getAdditionalAttribute` to closer match the JavaRosa API. I guess an alternative would be to rename/alias in JavaRosa as "additional" doesn't feel very clear here.

Reply:
Done.